### ISSUE TITLE
Build fails on a "blank" machine

### ISSUE THREAD

**Issue by arshad1990** [eclipse/xtext-web#51]
```
:org.eclipse.xtext.web:npmSetup SKIPPED
:org.eclipse.xtext.web:npmInstall
npm http GET https://registry.npmjs.org/mocha/2.2.5
npm http GET https://registry.npmjs.org/requirejs/2.1.18
npm http GET https://registry.npmjs.org/mocha/2.2.5
npm http GET https://registry.npmjs.org/requirejs/2.1.18
npm http GET https://registry.npmjs.org/mocha/2.2.5
npm http GET https://registry.npmjs.org/requirejs/2.1.18
npm ERR! Error: CERT_UNTRUSTED
npm ERR!     at TLSSocket.<anonymous> (_tls_wrap.js:777:36)
npm ERR!     at TLSSocket.EventEmitter.emit (events.js:98:17)
npm ERR!     at TLSSocket._finishInit (_tls_wrap.js:337:8)
npm ERR! If you need help, you may report this *entire* log,
npm ERR! including the npm and node versions, at:
npm ERR!     <http://github.com/isaacs/npm/issues>

npm ERR! System Darwin 17.3.0
npm ERR! command "/Users/aadavani/xtext-master/git/xtext-web/org.eclipse.xtext.web/.gradle/nodejs/node-v0.11.10-darwin-x64/bin/node" "/Users/aadavani/xtext-master/git/xtext-web/org.eclipse.xtext.web/.gradle/nodejs/node-v0.11.10-darwin-x64/bin/npm" "install"
npm ERR! cwd /Users/aadavani/xtext-master/git/xtext-web/org.eclipse.xtext.web
npm ERR! node -v v0.11.10
npm ERR! npm -v 1.3.22
npm ERR! 
npm ERR! Additional logging details can be found in:
npm ERR!     /Users/aadavani/xtext-master/git/xtext-web/org.eclipse.xtext.web/npm-debug.log
npm ERR! not ok code 0
:org.eclipse.xtext.web:npmInstall FAILED
```

**Comment by cdietrich** [comment]
i saw this yesterday on my machine too.
is it possible this is caused by an outdated node/npm version?
@holgerschill do you have any idea

**Comment by szarnekow** [comment]
Yes, you should certainly try the build with a more recent version, e.g. the node lts

**Comment by cdietrich** [comment]
yes see pull request

**Comment by cdietrich** [comment]
Fixed in 2.14
